### DISCUSSION TITLE
I want to add nlohmann (nlohmann-json-3.7.3.tar.gz) as third party library in my project, could someone tell how do i add this in my cmake file?

### DISCUSSION THREAD

**Discussion by Mangesh19** [nlohmann/json#3219] [Q&A] [answered]
I want to add nlohmann (nlohmann-json-3.7.3.tar.gz) as third party library in my project, could someone tell how do i add this in my cmake file?

**Comment by nlohmann** [comment, marked as answer]
You could use FetchContent as described here: https://json.nlohmann.me/integration/cmake/#embedded-fetchcontent

**Reply by Mangesh19** [reply]
Thanks this way is working...just would like to know how we can add nlohmann_json to implement using find_package?

**Comment by Mangesh19** [comment]
How should i create .so file from nlohmann-json-3.7.3.tar.gz using cmake?

**Reply by nlohmann** [reply]
The library is header-only - you cannot create a dynamic library from it, but you must compile it with your project's sources.

**Reply by nlohmann** [reply]
Do you need further assistance with this issue?
